bloom-housing/bloom
# Prisma change over feature branch

This PR addresses **PRISMA**


## Description

This PR pulls over the Prisma feature branch into the main branch. It's a doozy, but has all of the changes for the new backend along with the corresponding front-end changes.

*Note*: As of writing this there are two open PRs that have the remaining bug fixes which will automatically get added to this PR when they are merged.

## How Can This Be Tested/Reviewed?

This has been tested via all of the PRs during the Prisma development process along with the bug bashes.

However a quick check of pull it down and spinning it up wouldn't hurt.

## Checklist:

- [] I have made corresponding changes to the documentation

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
